common/test/acceptance/tests/studio/test_import_export.py

```diff
@@ -12,6 +12,25 @@
 
 
 # pylint: disable=no-member
+class TemplateTestMixin(object):
```

Comment:
@martynjames We do need to get this merged today, if possible. As your other comment was removed, and this one remains-- are there any specific changes you'd like me to make before giving the thumbs up? I know it's a little verbose, but if you have a preferred alternative approach, I can try getting it in today. I just need to get started now, if so.